activities/communication/blogging.md

```diff
@@ -0,0 +1,98 @@
+---
+Type: guide
+Explains: how to blog
+---
+
+# How and why we blog
+
+[We blog](https://blog.publiccode.net/) for a number of different reasons. This guide gives background information about the blog and instructions on how to blog.
+
+## Why we have a blog
+
+The blog is our primary tool for talking about:
+
+* our work: what we’re working on and what we’ve learned
+* our news: what we’ve achieved and what we’ll be doing next
+* our community: interviews with and guest posts by community members
+* our position on important issues: why we’ve made the political, technological, organizational or design decisions we have
+
+Not all information published on the blog needs to support our marketing strategy - for example, an in-depth discussion about a technology choice we’ve made may not persuade potential Foundation members, but will be interesting to technologists in our community.
+
+To make sure we have the editorial freedom to publish about all of our work (no matter how niche), there’s no direct link to the blog on [publiccode.net](https://publiccode.net/). Instead, we may feature important blogposts. (We’ll build this functionality once we have important blogposts.)
```

Comment:
We can also state clearly that opinions of authors are independent from the Foundation for Public Code. Also, if blog was linked from the website, search engines would identify it easier as our main Communications tool.

Reply:
Can you raise this as an issue on the blog?